dataflow/data/bigquery/pipeline_config.json

```diff
@@ -4,7 +4,7 @@
         "schemaFile": "./data/bigquery/schemas/base_downloads_ip.json",
         "outputTable": "data_viz.base_downloads_ip_by_day",
         "startDateFromTable": "data_viz.base_downloads_ip_by_day",
-        "ndtTable": "`measurement-lab.release.ndt_all`",
+        "ndtTable": "`mlab-oti.base_tables.ndt`",
```

Comment:
@pboothe @bartlbalazs Is this the correct dataset? I understand that we may have been testing `mlab-oti.base_tables.ndt`, but I was under the impression that if anything this might be `measurement-lab.ndt.base`.

Reply:
@gfr10598 for the authoritative answer

Reply:
Thanks @pboothe. Chatted with @gfr10598 in slack, but adding that answer here for future reference:
* the daily ingestion goes to `measurement-lab.base_tables`, 
* annotation reprocessing output goes to `mlab-oti.base_tables`
* daily ingestion tables will have duplicates and outdated annotations, 
* annotated tables are de-duplicated and fully annotated but up to two weeks behind daily ingestion tables